tests/integration/test_workspace_integration.py

```diff
@@ -0,0 +1,837 @@
+"""
+Integration tests for MLOps Stacks against Databricks workspaces.
+These tests initialize projects and deploy/run resources to validate end-to-end functionality.
+
+NOTE: These tests require Databricks workspace access and are marked as 'large'
+to be run only when explicitly requested with pytest --large.
+"""
+
+import json
+import os
+import pytest
+import subprocess
+import tempfile
+import time
+from pathlib import Path
+from utils import (
+    generate,
+    databricks_cli,
+)
+
+
+@pytest.fixture(scope="session")
+def current_user(databricks_cli, workspace_config):
+    """Get current user information once per session."""
+    user_result = subprocess.run(
+        [
+            databricks_cli,
+            "--profile",
+            workspace_config["profile"],
+            "current-user",
+            "me",
+            "--output",
+            "json",
+        ],
+        capture_output=True,
+        text=True,
+        timeout=10,
+    )
+
+    if user_result.returncode == 0:
+        user_info = json.loads(user_result.stdout)
+        return {
+            "username": user_info.get("userName", "unknown"),
+            "display_name": user_info.get("displayName", "unknown"),
+            "user_info": user_info,
+        }
+    else:
+        return {"username": "unknown", "display_name": "unknown", "user_info": {}}
+
+
+@pytest.fixture(scope="session")
+def workspace_config():
+    """
+    Get workspace configuration for integration tests.
+
+    Required environment variables:
+    - DATABRICKS_CONFIG_PROFILE: Databricks CLI profile name
+    - DATABRICKS_CLOUD: Cloud provider (aws, azure, or gcp)
+
+    Optional environment variables:
+    - TEST_CATALOG_NAME: Unity Catalog name for tests (default: 'test')
+    - TEST_SCHEMA_NAME: Schema name for tests (default: 'mlops_stacks_integration_tests')
+    """
+    profile = os.getenv("DATABRICKS_CONFIG_PROFILE")
+    cloud = os.getenv("DATABRICKS_CLOUD")
+
+    if not profile or not cloud:
+        pytest.skip(
+            "Integration tests require DATABRICKS_CONFIG_PROFILE and DATABRICKS_CLOUD environment variables. "
+            "Example: DATABRICKS_CONFIG_PROFILE=e2demo-fe-aws DATABRICKS_CLOUD=aws"
+        )
+
+    config = {
+        "profile": profile,
+        "cloud": cloud,
+        "catalog": os.getenv("TEST_CATALOG_NAME", "test"),
+        "schema": os.getenv("TEST_SCHEMA_NAME", "mlops_stacks_integration_tests"),
+    }
+
+    return config
+
+
+def _cleanup_unity_catalog_model(databricks_cli, workspace_config, project_name):
+    """Clean up Unity Catalog models by finding and deleting all matching models."""
```

Comment:
Why don't we just use the Databricks SDK instead of running all these subprocesses?